impl-tinkerpop/src/test/java/org/hawkular/inventory/impl/tinkerpop/test/BasicTest.java

```diff
@@ -283,8 +295,12 @@ public void testRelationshipServiceGet() throws Exception {
             "going from  environments must have uid equal to 'test'. Was: " + contains.getTarget().getId();
         assert "contains" .equals(contains.getName()) : "Name of the relation must be 'contains'.";
 
-        contains = inventory.tenants().getAll().environments().get("test").relationships().get("13").entity();
-        assert contains == null : "There should not be any edge with id 13.";
+        try {
+            inventory.tenants().getAll().environments().get("test").relationships().get("13").entity();
```

Comment:
13 is the magic number ;) It's mentioned in the jdoc, 

//here we assume the edge IDs in advance to be 10 and 14 and there is not edge with id 13

The underlying graph creates the IDs deterministically and these numbers are just assumed for the graph that is set up in the `setupData` method. Not super smart, I admit. Perhaps better strategy would be to fetch some relationship by some other criteria, find out the id and then fetch it by id and compare the two.